package.json

```diff
@@ -121,7 +121,10 @@
       "underscore": "1.12.1",
       "url-parse": "^1.5.0",
       "word-wrap": "^1.2.4",
-      "y18n": "^4.0.1"
+      "y18n": "^4.0.1",
+      "nth-check@<2.0.1": ">=2.0.1",
+      "@azure/identity@<4.2.1": ">=4.2.1",
+      "fast-xml-parser@<4.4.1": ">=4.4.1"
```

Comment:
Can you add explanation for these to the section below, so we know why they are here and when it would be safe to remove them?

Reply:
This was auto generated. I'm currently looking into seeing if I can just upgrade a single package that has these as a dependency to forgo needing the overrides.